webpack.config.babel.js

```diff
@@ -102,6 +104,8 @@ const TEST_CONFIG = { ...PROD_CONFIG, PUBLIC_PATH: '/', 'MOBILE_URL' : '/' }
 const STAGING_CONFIG = {
   'ONFIDO_API_URL': 'https://apidev.onfido.com',
   'ONFIDO_SDK_URL': 'https://sdk-staging.onfido.com',
+  'ONFIDO_TERMS_URL': 'https://dev.onfido.com/termofuse',
```

Comment:
oops sorry my bad it should be /termsofuse (I realise I mistyped termofuse in the ticket!)